workspace/busWebsite.js

```diff
@@ -0,0 +1,9 @@
+function clickFunction(){
+    var postcode = document.getElementById('postcode').value;
+    var xhttp = new XMLHttpRequest();
+    xhttp.open("GET", "http://localhost:3000/closestStops?postcode=" + postcode, false);
```

Comment:
You could tinker with making this asynchronous by setting `false` to `true` here (but I'm not going to make you, because it makes this _quite_ complicated.

(This is fine as it is, it would just be an interesting exercise to see how that worked!)